test/livebook_web/live/session_live_test.exs

```diff
@@ -36,6 +36,9 @@ defmodule LivebookWeb.SessionLiveTest do
       Session.set_notebook_name(session.pid, "My notebook")
       wait_for_session_update(session.pid)
 
+      # Wait for LV to update
+      render(view)
+
       assert page_title(view) =~ "My notebook"
```

Comment:
This test failed a couple times on the CI, while above we have similar test with render and it works fine, so I just added that to delay the title check.